components/auth/Join.tsx

```diff
@@ -92,6 +95,21 @@ const Join = () => {
           error={formik.touched.password ? formik.errors.password : undefined}
           onChange={formik.handleChange}
         />
+
+
+        {/* ADDING CHECKBOX HERE */}
+        <InputWithCheckbox
+          type="checkbox"
+          label={t('termsandconditon')}
+          name="agreeToTerms"
+          placeholder={t('termsandcondition')}
+          value={formik.values.agreeToTerms}
```

Comment:
Can you display the error message `You must agree to the Terms and Conditions.` below the checkbox?